### ISSUE TITLE
pyhelm is not reading requirements.yaml for the dependencies.

### ISSUE THREAD

**Issue by vijay-v6** [flaper87/pyhelm#52]
pyhelm is not reading requirements.yaml for the dependencies... Problem here is that pyhelm is not suitable for requirements.yaml having condition for each dependency which is automatically checked during helm installation.

**Comment by yanivoliver** [comment]
You're right, it's currently not supported by PyHelm. I'd be happy to go over your PR if you wish to add such capabilities.

**Comment by Prodian0013** [comment]
we just ran into this same issue.  will this issue be resolved soon?
